crates/bevy_pbr/src/atmosphere/aerial_view_lut.wgsl

```diff
@@ -32,7 +32,10 @@ fn main(@builtin(global_invocation_id) idx: vec3<u32>) {
 
     for (var slice_i: u32 = 0; slice_i < settings.aerial_view_lut_size.z; slice_i++) {
         for (var step_i: u32 = 0; step_i < settings.aerial_view_lut_samples; step_i++) {
-            let t_i = t_max * (f32(slice_i) + ((f32(step_i) + 0.5) / f32(settings.aerial_view_lut_samples))) / f32(settings.aerial_view_lut_size.z);
+            // Offset by -0.5 to align sampling position with slice boundaries, 
+            // since each texel stores the integral over its entire slice
```

Comment:
I added this offset here and comment based on the discussion on discord, I think I am happy with this version overall.

Reply:
![image](https://github.com/user-attachments/assets/fd2c85ac-a252-4ae8-9d2e-652e69ce718c)
for a bit more of a visual explanation as well, in this photo the red represents the center of each slice while the green is the boundaries in the texture coordinate space.